workflows/treeval.nf

```diff
@@ -256,7 +256,7 @@ workflow TREEVAL {
         REPEAT_DENSITY.out.repeat_density,
         params.entry
     )
-    ch_versions     = ch_versions.mix( HIC_MAPPING.out.versions )
+    ch_versions     = ch_versions.mix( HIC_MAPPING.out.versions )*/
```

Comment:
@yumisims . I don't see a matching `/*` for this one. I think it's this `*` that raises the error about "multiply"

Reply:
ah, thanks